src/commands/convert/hdf2caat.ts

```diff
@@ -0,0 +1,27 @@
+import {Command, Flags} from '@oclif/core'
+import fs from 'fs'
+import path from 'path'
+import {FromHDFToCAATMapper as Mapper} from '@mitre/hdf-converters'
+
+export default class HDF2CAAT extends Command {
+  static usage = 'convert hdf2caat -i <hdf-scan-results-json>... -o <output-caat-xlsx> [-h]'
+
+  static description = 'Translate an HDF file into a CAAT XLSX'
+
+  static examples = ['saf convert hdf2caat -i hdf_input.json -o caat-results.xlsx']
+
+  static flags = {
+    help: Flags.help({char: 'h'}),
+    input: Flags.string({char: 'i', required: true, multiple: true, description: 'Input HDF JSON file'}),
+    output: Flags.string({char: 'o', required: true, description: 'Output CAAT XLSX File'}),
```

Comment:
In no way trying to grow scope here, just curious. Do we usually or ever allow output to be optional and default to the CLI? I imagine that is typically not desirable, but just wondered.

Reply:
i've done it for some usecases where the intention is to immediately modify the structured content and pipe it back into the saf cli.  it'd be a bit of work but we could probably add it to the mappers as well if we wanted to, but i've not found anyone need to pipe raw file content around.  can you come up with a usecase where this would be useful?

https://github.com/mitre/saf/blob/main/src/commands/supplement/target/read.ts + write.ts
https://github.com/mitre/saf#target - example command provided with the piping